Build error

I am following the instruction [here](https://cvc5.github.io/docs/cvc5-1.0.0/installation/installation.html) to build `cvc5`, but when I trigger `make` command after running `./configure.sh --auto-download` successfully, something went error as follows. 

```
[ 92%] Building CXX object src/CMakeFiles/cvc5-obj.dir/util/statistics_public.cpp.o
/home/chenjiacheng/tmp/cvc5/src/util/real_algebraic_number_poly_imp.cpp: In member function ‘cvc5::internal::RealAlgebraicNumber cvc5::internal::RealAlgebraicNumber::operator/(const cvc5::internal::RealAlgebraicNumber&) const’:
/home/chenjiacheng/tmp/cvc5/src/util/real_algebraic_number_poly_imp.cpp:272:33: error: no match for ‘operator/’ (operand types are ‘poly::AlgebraicNumber’ and ‘poly::AlgebraicNumber’)
  272 |     return convertToPoly(*this) / convertToPoly(rhs);
      |            ~~~~~~~~~~~~~~~~~~~~ ^ ~~~~~~~~~~~~~~~~~~
      |                         |                      |
      |                         poly::AlgebraicNumber  poly::AlgebraicNumber
```
The error message confused me and I don't know how to fix it.
Plus: the operating system I am using is `Ubuntu 20.04.6 LTS`
Thanks a lot if you could give me some instruction!

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -473,7 +473,7 @@ if(USE_KISSAT)
 endif()
 
 if(USE_POLY)
-  find_package(Poly REQUIRED)
+  find_package(Poly 0.1.13 REQUIRED)
   add_definitions(-DCVC5_USE_POLY)
   set(CVC5_USE_POLY_IMP 1)
 else()

--- a/INSTALL.rst
+++ b/INSTALL.rst
@@ -175,8 +175,8 @@ may improve performance. It can be downloaded and built automatically. Configure
 cvc5 with ``configure.sh --kissat`` to build with this dependency.
 
 
-LibPoly (Optional polynomial library)
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+LibPoly >= v0.1.13 (Optional polynomial library)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 `LibPoly <https://github.com/SRI-CSL/libpoly>`_ is required for CAD-based
 nonlinear reasoning. It can be downloaded and built automatically. Configure

